packages/editor-ui/src/stores/pushConnection.store.ts

```diff
@@ -142,7 +142,8 @@ export const usePushConnectionStore = defineStore(STORES.PUSH, () => {
 	async function pushMessageReceived(event: Event) {
 		let receivedData: PushMessage;
 		try {
-			// @ts-ignore
+			// @Cleanup: Check for data explicitly and log telemetry on missing field if appropriate
```

Comment:
We should reconsider adopting this pattern of comments. I don't think this type of comments is a good idea, because often these comments end up living long after, never getting fixed, or worse, being leftover even if core issue was fixed. 
If the goal is simply to create Linear tickets, we should find an alternative way, like doing it manually or automatically off of this branch, but I don't recommend we merge this. I could see this getting adopted and picked up by other engineers as a lazy way to avoid a refactor.

Reply:
Thanks for the feedback! You're definitely right in that we should get further input if we introduce this pattern at this scale, I was a bit too hasty.

Reply:
Considering the amount of `TODO` tasks we already have this repo, I have doubts about if these additional `@Cleanup` or `@Problem` would get addressed any time soon.
I'm fine with these comments, but only if we also create linear tickets for these and add a link to those here. That way those tickets can be assigned and prioritized.

Reply:
How about I create the workflow to track these before we merge this? That way we're committed. Would that work @mutdmour ?

I really think this is useful beyond reducing warnings in the repo, I've already discovered a few potential bugs just in this first batch.

Reply:
Gotcha, that makes a lot of sense 👍 As long as we track this work somewhere, we have a plan to address it and it is clear what the prio of each one is, I'm all up for it